src/api/apiUtils.js

```diff
@@ -42,10 +42,10 @@ export function getOrganizationUUID() {
 
 export const makeQueryString = (filterObj) => {
   // log.debug('makeQueryString 1 ----->', filterObj);
-  const arr = [];
```

Comment:
Sorry @gwynndp, I'm being an idiot. You were right to make these `const`.
I'm still getting up to speed with all this modern JS stuff 🤣 

I'll change it back.